templates/cakephp/@layout.latte

```diff
@@ -24,7 +24,7 @@ the file LICENSE.md that was distributed with this source code.
 	<link rel="icon" type="image/png" sizes="32x32" href="{='resources/favicons/favicon-32x32.png'|staticFile}">
 	<link rel="icon" type="image/png" sizes="16x16" href="{='resources/favicons/favicon-16x16.png'|staticFile}">
 	<link rel="shortcut icon" type="image/x-icon" href="{='resources/favicons/favicon.ico'|staticFile}"/>
-	<link rel="manifest" href="{='resources/favicons/manifest.json'|staticFile}">
+	<link rel="manifest" href="{='resources/favicons/manifest.json'}">
```

Comment:
Why is this change necessary?

Reply:
Need to make the path without extra characters
https://realfavicongenerator.net/favicon_checker?protocol=https&site=api.cakephp.org%2F2.6%2F#.Yi01pXrP23A

![Screenshot_1](https://user-images.githubusercontent.com/26163841/158039358-42bc5778-57e6-484b-acff-ef5d81231e32.png)